script/Arcueid/Day 1/01_00_ARC01_6_1.txt

```diff
@@ -235,7 +235,7 @@ Freedom... no, egotism would describe it better. These are rights that you can o
 -- Page 322, Offset 1979.
 -- 「君は遠野の子でありながら、それを理解していなかった。秋葉とは似ても似つかぬ三流だ。まあ、一度地に落ちていながら帰ってくる厚顔さだけは、一流と言えなくもないがね」
 "You are a son of the Tohno family, but you didn't understand that.
- While similar to Akiha at first glance, you're really sub-par.
+ While similar to Akiha at first glance, you're really subpar.
  Well, your nerve is up to par at least, seeing as how you came back here after falling so low."
```

Comment:
I'd probably prefer this construction so the other meaning of "really" doesn't cause a misinterpretation.
>...glance, you're subpar, really.